### PR TITLE
Adds the UI package

### PR DESCRIPTION
**Pull request by balugege**

### CODE REVIEW

File: packages/stubs/node/src/ts/storage/keyvaluestorage/FireBaseKeyValueStorage.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export class FireBaseKeyValueStore<T> implements KeyValueStorage<T> {
     }
 
     public async put(key: string, value: unknown): Promise<Reply> {
-        const document = this.client.doc('keyValueStorage/' + key);
+        const document = this.client.doc('eyValueStorage/' + key);
```
Comment (balugege) [inline packages/stubs/node/src/ts/storage/keyvaluestorage/FireBaseKeyValueStorage.ts:20]: Was this change intentional?

Comment (eliasgroll) [reply packages/stubs/node/src/ts/storage/keyvaluestorage/FireBaseKeyValueStorage.ts:20]: nope, though van we add the doc as a config parameter?

Comment (balugege) [reply packages/stubs/node/src/ts/storage/keyvaluestorage/FireBaseKeyValueStorage.ts:20]: I will revert it. You are right that it should be passed as a config parameter, but that is not inside the scope of this PR.

**Review by eliasgroll — APPROVED** [review]
